Classes/New Issue/NewIssueTableViewController.swift

```diff
@@ -213,11 +225,18 @@ final class NewIssueTableViewController: UITableViewController, UITextFieldDeleg
         return false
     }
 
+    // MARK: UITextViewDelegate
+
+    /// Called when editing changed on the body field, enable/disable submit button based on title and body
```

Comment:
Awesome you thought about the header docs! If we drop the body requirement though, this should be updated.